### PR TITLE
Refactor code, update logo, improve SEO, and updated styling for footer and default page data container view

### PR DESCRIPTION
**Pull request by sejalkhatri**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "url-join": "^4.0.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "call env.bat && react-scripts start",
```
Comment (notconfusing) [inline package.json]: call env.bat
then
react-scripts-start

File: src/App.css [hunk 2]
```diff
@@ -84,7 +84,6 @@ svg {
 
 .footer {
   background: #F3F0F0;
-  position: -webkit-sticky;
```
Comment (notconfusing) [inline src/App.css:94]: Just double checking, we are removing the stick footer?
I kind of like that decision.

File: package.json (hunk 1 above)
Comment (notconfusing) [reply package.json]: Lets remove this modification.

File: src/Components/FAQ.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		</Accordion>
+    </div>
+  )
+
+
+}
+
+export default FAQ
+
+    // <Navbar sticky="bottom" className="footer">
```
Comment (notconfusing) [inline src/Components/FAQ.js]: delete this cruft

File: src/Components/FAQ.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import Accordion from 'react-bootstrap/Accordion'
+import Card from 'react-bootstrap/Card'
+import Button from 'react-bootstrap/Button'
+import { useAccordionToggle } from 'react-bootstrap/AccordionToggle';
+
+function FAQ(){
+
+  return (
+    <div>
```
Comment (notconfusing) [inline src/Components/FAQ.js]: put Id on this Div

File: src/Components/Footer.js [hunk 5]
```diff
@@ -3,9 +3,39 @@ import  {Navbar}  from 'react-bootstrap'
 
 function Footer(){
   return (
-    <Navbar sticky="bottom" className="footer">
-      Powered by a grant funded by the Wikimedia Foundation 
-    </Navbar>
+    <footer>
+   		<div className="flex-container">
+  			<div></div>
```
Comment (notconfusing) [inline src/Components/Footer.js]: are these empty divs important?

File: src/Components/Footer.js [hunk 6]
```diff
@@ -3,9 +3,39 @@ import  {Navbar}  from 'react-bootstrap'
...
+ 	 				<li><a href="">Humaniki API</a></li>
+ 	 				<li><a href="">FAQ</a></li>
+ 	 			</ul>
+ 	 		</section>
+ 	 		<section>
+ 	 			<h5> Contributing </h5> 
+ 	 			<ul> 
+ 	 				<li><a href="">Phabricator</a></li>
+ 	 				<li><a href="">Github</a></li>
+ 	 				<li><a href="">IRC</a></li>
```
Comment (notconfusing) [inline src/Components/Footer.js]: Is that going to be our element channel?

File: src/Components/Footer.js [hunk 7]
```diff
@@ -3,9 +3,39 @@ import  {Navbar}  from 'react-bootstrap'
...
+ 	 				<li><a href="">About Humaniki</a></li>
+ 	 				<li><a href="">Humaniki API</a></li>
+ 	 				<li><a href="">FAQ</a></li>
+ 	 			</ul>
+ 	 		</section>
+ 	 		<section>
+ 	 			<h5> Contributing </h5> 
+ 	 			<ul> 
+ 	 				<li><a href="">Phabricator</a></li>
+ 	 				<li><a href="">Github</a></li>
```
Comment (notconfusing) [inline src/Components/Footer.js]: there are mutliple github repos, lets think about how to show that?

File: src/Views/AboutView.js [hunk 8]
```diff
@@ -9,12 +10,14 @@ function AboutView(){
     <Container className="About">
       <div className="About-Content">
         <h3>We create awareness about content diversity in online knowledge spaces </h3>
-        <img 
-          className="about-img"
-          src="./politiciansGenderGap.png" 
-          alt="world leaders walking and talking from the 37th G8 summit" 
-        ></img>        
-        <h5>The World Leaders</h5>
+        <figure>
```
Comment (notconfusing) [inline src/Views/AboutView.js:12]: NICE, very elegant.

File: src/Views/DefaultView.js [hunk 9]
```diff
@@ -76,21 +77,14 @@ function DefaultView({API}){
...
-          date of birth, place of birth, occupation, and language of biography articles 
-          in all Wikipedias.
+          For example, as of August 2020, only 17% of content in all Wikimedia projects including biographies on Wikipedia are about women.
         </h5>
       </Row>
       {isLoading ? loadingDiv : null }
       {isErrored ? errorDiv : null }
       {!isLoading && !isErrored ? viz : null }
-      <Row className="About-Explainer">
-        Expore further dynames of the gender gap in bibliographic content on Wikipedia
```
Comment (notconfusing) [inline src/Views/DefaultView.js:90]: typo "dynames"

**Review by notconfusing — COMMENTED** [review]
Goold work @sejalkhatri 

image

File: src/Components/Footer.js (hunk 6 above)
Comment (sejalkhatri) [reply src/Components/Footer.js]: yes! Do we plan on keeping it active or not? 

If active then which configuration should we enable? 
option 1: Anyone who knows the room's link, apart from guests
option 2: Anyone who knows the room's link, including guests

If not active then we can direct to our discussion page on MediaWiki - https://www.mediawiki.org/wiki/Talk:Humaniki
or, 
on Phabricator

File: src/Components/Footer.js (hunk 7 above)
Comment (sejalkhatri) [reply src/Components/Footer.js]: We can redirect this link to Readme on Github (either Eugenia's repo or yours) where we provide information about how we maintain 2 repo for backend and frontend, and provide links!

File: src/App.css (hunk 2 above)
Comment (sejalkhatri) [reply src/App.css:94]: Having a sticky footer is not the same as keeping the position sticky. I am looking for options to achieve that. 

An element with position: sticky; is positioned based on the user's scroll position. A sticky element toggles between relative and fixed , depending on the scroll position. It is positioned relative until a given offset position is met in the viewport - then it "sticks" in place (like position:fixed).

File: src/Views/DefaultView.js [hunk 10]
```diff
@@ -62,36 +64,49 @@ function DefaultView({API}){
   const loadingDiv = <div>Loading</div>
   const viz = 
     <div className="default-data-container">
-      <h4> Recent Distribution of Articles </h4>
-        <h3> {totalMen} Male Biographies </h3>
-        <h3> {totalOthers} Σ Other Biographies </h3>
-        <h3> {totalWomen} Female Biographies </h3>
+      <h5> Global Gender Gap </h5>
+      <h6> Disctrribution of content of humans in all Wikimedia Projects </h6>
```
Comment (TheEugeniaKim) [inline src/Views/DefaultView.js]: fix typo and (sum) to end (line 76)

File: src/Views/DefaultView.js [hunk 11]
```diff
@@ -62,36 +64,49 @@ function DefaultView({API}){
...
+          <h4> {totalWomen} </h4>
+          <h6> Female Biographies </h6>
+        </div>
+      </div>
       <SingleBarChart genderTotals={[
         (totalMen/total*100), 
         (totalOthers/total*100), 
         (totalWomen/total*100)
       ]} />
+      <p>All time, as of Aug'20</p>
```
Comment (TheEugeniaKim) [inline src/Views/DefaultView.js]: LATEST SNAPSHOT

File: src/Views/DefaultView.js [hunk 12]
```diff
@@ -62,36 +64,49 @@ function DefaultView({API}){
...
-        with Humaniki and learn how you can contribute to bridge this gap. Compare gender 
-        diversity across Wikipedia language editions, gender by country, and date of birth. 
-      </Row>
-    </Container>
+    <div className="default-main">
+      <div className="default">
+        <Row className="default-content">
+          <h4 className="default-title">Humaniki provides statistics about the gender gap in the content of all Wikimedia projects</h4>
+          <h6>
+            For example, as of August 2020, only 17% of content in all Wikimedia projects including biographies on Wikipedia are about women.
```
Comment (TheEugeniaKim) [inline src/Views/DefaultView.js]: LATEST SNAPSHOT (LATEST TOTAL COVERAGE)

**Review by TheEugeniaKim — CHANGES_REQUESTED** [review]
Looks good. Just need to resolve the container issue

**Review by TheEugeniaKim — APPROVED** [review]

**Pull request merged**